### PR TITLE
[W4.11e][T10-1]Pang Jia Hao

### PR DESCRIPTION
**Pull request by pangjiahao**
UtilsTest.java: add test for isAnyNull method

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +36,27 @@ public void elementsAreUnique() throws Exception {
...
+        assertIsAnyNull(1, 2, null, 3);
+        assertIsAnyNull(1, 2, null, 3, null, null, 4, 5);
+
+        // no null objects
+        assertNotIsAnyNull();
+        assertNotIsAnyNull(1);
+        assertNotIsAnyNull(1, "hello", 3, "null", 4);
+    }
+
+    private void assertIsAnyNull(Object... objects) {
```
Comment (yunpengn) [inline test/java/seedu/addressbook/common/UtilsTest.java:52]: Good idea on using abstraction and extracting methods. This is generally a good practice, not that essential in this case though.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -36,6 +36,27 @@ public void elementsAreUnique() throws Exception {
...
+    public void testIsAnyNull() {
+        // 1 or more null objects
+        assertIsAnyNull((Object) null);
+        assertIsAnyNull(1, 2, null, 3);
+        assertIsAnyNull(1, 2, null, 3, null, null, 4, 5);
+
+        // no null objects
+        assertNotIsAnyNull();
+        assertNotIsAnyNull(1);
+        assertNotIsAnyNull(1, "hello", 3, "null", 4);
```
Comment (yunpengn) [inline test/java/seedu/addressbook/common/UtilsTest.java:49]: Enough test cases. Great!

**Review by yunpengn — COMMENTED** [review]
Please close this PR after viewing my comments.

File: test/java/seedu/addressbook/common/UtilsTest.java (hunk 2 above)
Comment (pangjiahao) [reply test/java/seedu/addressbook/common/UtilsTest.java:49]: Thanks Yunpeng!

**Pull request closed without merging**